packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,274 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Interact with image and video. It lets the app developer ask the user to get images or video from their camera / camera roll / file system.
+ * @beta
+ */
+export namespace visualMedia {
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * All properties common to Image and Video Props
+   * @beta
+   */
+  interface VisualMediaProps {
+    /**
+     * @hidden
+     * Lets the developer specify the media source
+     */
+    source: Source;
+
+    /**
+     * @hidden
+     * Optional; Specify in which mode the camera will be opened.
+     * Default value is Photo
+     */
+    startMode?: CameraStartMode;
+
+    /**
+     * @hidden
+     * Optional; indicate if user is allowed to move between front and back camera
+     * Default value is true
+     */
+    cameraSwitcher?: boolean;
+    /**
+     * max limit of media allowed to be selected in one go, current max limit is 10 set by lens-sdk.
+     */
+    visualMediaCount: number;
+  }
+  /**
+   *  All properties in ImageProps are optional and have default values in the platform
+   *  Additional properties for image in mobile devices
+   * @beta
+   */
+  export interface ImageProperties extends VisualMediaProps {
+    /**
+     * Optional; indicate if inking on the selected Image is allowed or not
+     * Default value is true
+     */
+    ink?: boolean;
```

Comment:
>ink [](http://example.com/codeflow?start=4&length=3)

Since these are boolean values, can you prepend a verb to them to make their usage clearer? e.g. `shouldAllowInkingOnImages` and `shouldAllowTextStickersOnImages`

Reply:
Updated to shouldAllowInkingOnImages and shouldAllowTextStickersOnImages. Thanks for the naming suggestions.